src/common/utilscmn.cpp

```diff
@@ -1517,24 +1517,40 @@ wxWindowDisabler::wxWindowDisabler(bool disable)
 {
     m_disabled = disable;
     if ( disable )
+    {
         DoDisable();
+
+#if defined(__WXOSX__) && wxOSX_USE_COCOA
+        AfterDisable(NULL);
```

Comment:
@csomor I'm not sure if we should be doing this, this (may) end(s) up calling `wxGUIEventLoop::BeginModalSession(NULL)` and it's not clear to me if this is really desirable here. I've done it like this because this preserves the behaviour of the original code, but I don't know if it was really intentional.